redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/base/DatabaseServerV4Base.java

```diff
@@ -38,11 +38,15 @@
     @ApiModelProperty(value = DatabaseServer.PORT, required = true)
     private Integer port;
 
+    @NotNull
+    @ApiModelProperty(value = DatabaseServer.DATABASE_VENDOR, required = true)
+    private String databaseVendor;
+
     @ApiModelProperty(DatabaseServer.CONNECTOR_JAR_URL)
     private String connectorJarUrl;
 
-    // @ApiModelProperty(ModelDescriptions.ENVIRONMENTS)
-    // private Set<String> environments = new HashSet<>();
+    @ApiModelProperty(ModelDescriptions.ENVIRONMENT_IDS)
+    private Set<Long> environmentIds = new HashSet<>();
```

Comment:
1. Current info: 1 rds can be part of 1 env only.
2. We will have to store the CRN of the env

Reply:
OK - changing this to a simple String column then.